### DISCUSSION TITLE
데이터 관리

### DISCUSSION THREAD

**Discussion by jaeyeong815** [WantedPreonboardingFE5team/pre-onboarding-7th-2-1-5#41] [General]
혹시 다들 데이터 관리는 어떤걸 쓰나요? context API?
아니면 상태관리 라이브러리 사용하시나요? redux, recoil 등등..

**Comment by yeondooo** [comment]
recoil을 한번은 써보고 싶긴 했는데 말이죠....

**Reply by jaeyeong815** [reply]
ㅜㅜ.. 저는 뭘 써도 좋으니 제대로 됐으면 좋겠어요.. 타스 진짜..진짜,,ㅠㅠㅠ 컨텍스트 쓰려다가 없앴다가 지금은 axios도 지우고 fetch로 바꾸고 난리치는 중 임돠 미치겠슴다 다들 살아계시죠 화이팅입니😭
